src/ActionList/Item.tsx

```diff
@@ -13,13 +13,18 @@ export interface ItemProps extends React.ComponentPropsWithoutRef<'div'>, SxProp
   /**
    * Primary text which names an `Item`.
    */
-  text: string
+  text?: string
```

Comment:
I think it makes sense to keep `text` because of the complex rendering possible if you include `text` and `description` on the same item.  Seems like a case that could be fairly common, and it's already supported out of the box.

Reply:
I've always had the opinion that rendering should be kept separate from the text _data_. You need the raw text for things like keyboard navigation and filtering, but the rendering can be very complex and custom. Shouldn't text be required? Otherwise how can you u implement the "typeahead" pattern?

Reply:
Implement a default renderer that throws the provided text in a div or whatever, and use that if they don't provide children.